packages/config/src/projects/common/exits.ts

```diff
@@ -106,6 +106,33 @@ const STARKEX_EMERGENCY: ProjectTechnologyChoice = {
   ],
 }
 
+const STARKNET_REGULAR: ProjectTechnologyChoice = {
+  ...REGULAR('zk', 'no proof'),
+  description:
+    REGULAR('zk', 'no proof').description +
+    ' Note that the withdrawal request can be censored by the Sequencer.',
+  references: [
+    {
+      text: ' Withdrawing is based on l2 to l1 messages - StarkNet documentation',
+      href: 'https://www.cairo-lang.org/docs/hello_starknet/l1l2.html',
+    },
+  ],
+  //  risks: [RISK_CENTRALIZED_VALIDATOR],
+}
+
+const STARKNET_EMERGENCY: ProjectTechnologyChoice = {
+  ...EMERGENCY('', 'merkle proof'),
```

Comment:
The first argument shouldn't be an empty string. `EMERGENCY` is only a helper, you don't have to use it :)